requirements/env_omnisci.yml

```diff
@@ -14,3 +14,4 @@ dependencies:
   - omniscidbe4py
   - pip:
     - ray>=1.0.0
+    - s3fs>=0.4.2
```

Comment:
Why don't you install the package directly through the conda?

Reply:
I moved this above to dependencies. Did you mean that?

Reply:
yes